CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added ability to override the default password length
+
+### Changed
+- Changed the default password length for VNC sessions to be 8
```

Comment:
The default password length used to be 8, so I'd just remove the changed section here.